README.md

```diff
@@ -747,6 +748,97 @@ what the ideal dimensions are for the image they should upload.
   + `fw1` - "font weight 1", used to "fade" the font on the instructions.
   + `fw5` - "font weight 5", make the pixel dimensions more prominent.
 
+To view this in action,
+run
+`npm satart`
+and visit:
+http://127.0.0.1:8000/examples/image-placeholder.html
+
+
+## Responsive Navigation Menu
+
+Let's create a responsive navigation menu with _only_ HTML and CSS!
+
+This is what you can expect in the desktop view:
+
+![desktop-view-nav-menu](https://user-images.githubusercontent.com/194400/73554472-aa5de800-4443-11ea-941e-46c0d5c61337.png)
+
+And this is what it looks like in a constrained (_mobile_) view:
+
+![mobile-view-burger-nav-menu](https://user-images.githubusercontent.com/194400/73554480-ae8a0580-4443-11ea-9253-cf9d231c29c6.png)
+
+The _expanded_ burger menu:
+
+![mobile-view-burger-nav-menu](https://user-images.githubusercontent.com/194400/73554487-b2b62300-4443-11ea-981a-cf26e3818688.png)
+
+This CSS-only (_no `JavaScript` required!_) responsive navigation menu
+relies on [Isabel Castillo](https://github.com/isabelc)'s
+brilliant _hidden_ `checkbox` idea:
+https://isabelcastillo.com/pure-css-mobile-toggle-menu
+(shared by @iteles in
```

Comment:
Haha I didn't need the shout-out but thank you!

Reply:
you found the pearl. definitely worth recognising. recognition is core to our _culture_. ❤️